clients/githubrepo/search.go

```diff
@@ -59,7 +59,7 @@ func (handler *searchHandler) buildQuery(request clients.SearchRequest) (string,
 	}
 	var queryBuilder strings.Builder
 	if _, err := queryBuilder.WriteString(
-		fmt.Sprintf("%s repo:%s/%s", request.Query, handler.owner, handler.repo)); err != nil {
+		fmt.Sprintf("%s repo:%s/%s", strings.ReplaceAll(request.Query, "/", " "), handler.owner, handler.repo)); err != nil {
```

Comment:
why was there a `/` in the query that caused the issue? I thought org and repo did not have if they had been extracted properly from `owner/repo`?

Reply:
Ah, for fuzzing we check if the URI is in a `project.yaml`.

e.g. we search for `github.com/util-linux/util-linux` in the main_repo field of the YAML https://github.com/google/oss-fuzz/blob/56b840aad7907d69ad0de08a9339ba7e18a03514/projects/util-linux/project.yaml#L17